artifactory/services/download.go

```diff
@@ -133,7 +143,7 @@ func produceTasks(items []utils.ResultItem, downloadParams DownloadParams, produ
 		}
 		if v.Type != "folder" {
 			// Add a task, task is a function of type TaskFunc which later on will be executed by other go routine, the communication is done using channels.
-			// The second argument is a error handling func in case the taskFunc return an error.
+			// The second argument is an error handling func in case the taskFunc return an error.
 			tasksCount++
 			producer.AddTaskWithError(fileHandler(tempData), errorsQueue.AddError)
 			// We don't want to create directories which are created explicitly by download files when the --include-dirs flag is used.
```

Comment:
"...when the --include-dirs flag is used." --> "...when **ArtifactoryCommonParams.IncludeDirs** is used."

Reply:
Changed.